index.html

```diff
@@ -25,15 +25,20 @@
                 <a href="#main" class="skip">Skip to main content</a>
                 <nav class="flex-container">
                     <div class="logo-div">My Logo</div>
-                    <div class="hamburger_icon mobile"> <img src="img/hamburger-menu-icon.png" width="20">
+                    <div class="hamburger">
+                        <span class="bar"></span>
+                        <span class="bar"></span>
+                        <span class="bar"> </span>
                     </div>
-                    <div class="desktop">
+                    
+                    <!-- <div class="hamburger_icon mobile"> <img src="img/hamburger-menu-icon.png" width="20"> -->
+                    <!-- </div> -->
+                    
                         <ul class="navlink">
-                            <li>Portfolio</li>
-                            <li>About</li>
-                            <li>Contact</li>
+                            <li> <a href="#portfolioid"> Portfolio </a> </li>
+                            <li> <a href="#aboutid"> About </a> </li>
```

Comment:
- [ ] Kindly check if the `About` link is scrolling to the corresponding section when clicked. Currently, it is not doing anything when clicking on it. Please update it accordingly. 🤝️